app/pages/home-not-logged-in/research.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import counterpart from 'counterpart';
+import { Link } from 'react-router';
+
+counterpart.registerTranslations('en', {
+  researchHomePage: {
+    aboutIntro: `
+      A vibrant community.
+    `,
+    about: `
+      Zooniverse gives people of all ages and backgrounds the chance to paricipate
+      in real research with over 50- active online citizen science projects. Work with
+      1.5 million registered users around the world to contribute to research
+      projects led by hundreds of researchers.
+    `,
+    classifications: 'Classifications so far',
+    labs: 'Zooniverse Labs',
+    listen: `
+      The projects created with the Zooniverse project builder
+      inspire and empower researchers, designers, and artists
+    `,
+    meetResearchers: 'Meet the researchers who\'ve created projects for free on the Zooniverse.',
+    options: 'Sign in or register to get started',
+    real: 'Real researchers, real results',
+    researcherIntro: 'Meet the researchers who’ve created projects for free on the Zooniverse. ',
+    researcher: `
+      From classifying animals in the Serengeti to discovering new exoplanets using
+      the Kepler space telescope, researchers of all backgrounds have used the free
+      project builder to create engaging, accessible citizen science projects.
+      Our researchers have used the data from their projects to publish over 100
+      peer-reviewed publications that encourage many fascinating discoveries. Researchers
+      take part in project creation, data analysis, and even communicate directly
+      with volunteers through Zooniverse Talk.
+    `,
+    signIn: 'Sign in',
+    register: 'Register',
+    works: 'The Zooniverse Works'
+  }
+});
+
+const HomePageResearch = ({ count, screenWidth, showDialog }) => {
+  return (
+    <section className="home-research">
+      <Translate className="tertiary-kicker" content="researchHomePage.works" />
+      <h1 className="class-counter">{count.toLocaleString()}</h1>
```

Comment:
`h1` should only be used once on a page, as the very top level heading. Subsections should be headed with `h2`, `h3` etc. depending on the section nesting.